docs/PowerAuth-Server-Compatibility.md

```diff
@@ -0,0 +1,37 @@
+# PowerAuth Server Compatibility
+
+This document describes the compatibility between the PowerAuth Mobile Flutter SDK and PowerAuth Server versions.
+
+## Flutter SDK as a Wrapper
+
+The PowerAuth Mobile Flutter SDK is a wrapper built on top of the native PowerAuth Mobile SDK. It provides a Flutter-friendly interface while leveraging the proven cryptographic implementations and network communication protocols of the underlying native SDK.
+
+For more information about the native PowerAuth Mobile SDK, please refer to the [PowerAuth Mobile SDK repository](https://github.com/wultra/powerauth-mobile-sdk).
+
+## Current Version Compatibility
+
+The current version of the PowerAuth Mobile Flutter SDK includes:
+
+- **Native SDK Version**: 1.9.4
+- **Required Server Version**: 1.9.0 or higher
+
+### Version Details
+
+- **iOS**: Uses PowerAuth2 framework version 1.9.4
```

Comment:
Keep it simple, just say that iOS and Android are using the same version 1.9.4, and make the 1.9.4 bold and more prominent.